src/layouts/DatasetInfoLayout.astro

```diff
@@ -0,0 +1,88 @@
+---
+import field_map from "../data/field-names-mapping.json";
+import DatasetDetailLayout from "./DatasetDetailLayout.astro";
+
+const { eic } = Astro.props;
+const biosample_map = field_map["biosample"];
+const sipp_map = field_map["specimen_imaging_preparation_protocol"];
+const image_acquisition_map = field_map["image_acquisition"];
+const analysis_method_map = field_map["analysis_method"];
+---
+
+<div>
+    <div>
+        <b>{eic.title_id}</b>
+    </div>
+    <ul>
+        <div>
+            <b>Description:</b>&nbsp;{eic.description}
+        </div>
+        {
+            "specimen_imaging_preparation_protocol" in eic &&
+                eic.specimen_imaging_preparation_protocol.length > 0 && (
+                    <>
+                        <b>Biosamples:</b>
+                        <ul>
+                            {eic.biological_entity.map((biosample) => (
+                                <DatasetDetailLayout
+                                    data={biosample}
+                                    field_map={biosample_map}
+                                />
+                            ))}
+                        </ul>
+                    </>
+                )
+        }
+        {
+            "specimen_imaging_preparation_protocol" in eic &&
+                eic.specimen_imaging_preparation_protocol.length > 0 && (
+                    <>
+                        <b>Specimen Preparation Protocols:</b>
+                        <ul>
+                            {eic.specimen_imaging_preparation_protocol.map(
+                                (sipp) => (
+                                    <DatasetDetailLayout
+                                        data={sipp}
+                                        field_map={sipp_map}
+                                    />
+                                ),
+                            )}
+                        </ul>
+                    </>
+                )
+        }
+        {
```

Comment:
I think we need to add specimen_growth_protocol mapping here as well for completeness even though it's empty for this test study json.